src/octree/octree_iterator.rs

```diff
@@ -7,7 +7,7 @@ use collision::Aabb3;
 use std::collections::VecDeque;
 
 /// returns an Iterator over the points of the current node
-fn get_node_iterator(octree: &Octree, node_id: &NodeId) -> NodeIterator {
+pub fn get_node_iterator(octree: &Octree, node_id: &NodeId) -> NodeIterator {
```

Comment:
Why did this become public?

Reply:
here i forgot to remove some previous changes

Reply:
fixed and, most importantly, pushed to origin